docs/guides/server/logging.adoc

```diff
@@ -295,267 +304,14 @@ You can use regular expressions to exclude them, such as:
-
-This example abbreviates the category name to three characters by setting `[%c{3.}]` in the template instead of the default `[%c]`.
-
-=== Configuring the Syslog type
-
-Syslog uses different message formats based on particular RFC specifications.
-To change the Syslog type with a different message format, use the `--log-syslog-type` option as follows:
-
-<@kc.start parameters="--log-syslog-type=rfc3164"/>
-
-Possible values for the `--log-syslog-type` option are:
-
-<@opts.expectedValues option="log-syslog-type"/>
-
-The preferred Syslog type is https://datatracker.ietf.org/doc/html/rfc5424[RFC 5424], which obsoletes https://datatracker.ietf.org/doc/html/rfc3164[RFC 3164], known as BSD Syslog protocol.
-
-=== Configuring the Syslog maximum message length
-
-To set the maximum length of the message allowed to be sent (in bytes), use the `--log-syslog-max-length` option as follows:
-
-<@kc.start parameters="--log-syslog-max-length=1536"/>
-
-The length can be specified in memory size format with the appropriate suffix, like `1k` or `1K`.
-The length includes the header and the message.
-
-If the length is not explicitly set, the default values are set based on the `--log-syslog-type` option as follows:
-
-* `2048B` - for RFC 5424
-* `1024B` - for RFC 3164
-
-=== Configuring the Syslog structured output
-By default, the Syslog log handler sends plain unstructured data to the Syslog server.
-To use structured JSON log output instead, enter the following command:
-
-<@kc.start parameters="--log-syslog-output=json"/>
-
-.Example Log Message
-[source, bash]
-----
-2024-04-05T12:32:20.616+02:00 host keycloak 2788276 io.quarkus - {"timestamp":"2024-04-05T12:32:20.616208533+02:00","sequence":9948,"loggerClassName":"org.jboss.logging.Logger","loggerName":"io.quarkus","level":"INFO","message":"Profile prod activated. ","threadName":"main","threadId":1,"mdc":{},"ndc":"","hostName":"host","processName":"QuarkusEntryPoint","processId":2788276}
-----
-
-When using JSON output, colors are disabled and the format settings set by `--log-syslog-format` will not apply.
-
-To use unstructured logging, enter the following command:
-
-<@kc.start parameters="--log-syslog-output=default"/>
-
-.Example Log Message
-[source, bash]
-----
-2024-04-05T12:31:38.473+02:00 host keycloak 2787568 io.quarkus - 2024-04-05 12:31:38,473 INFO  [io.quarkus] (main) Profile prod activated.
-----
-
-As you can see, the timestamp is present twice, so you can amend it correspondingly via the `--log-syslog-format` property.
-
-<@opts.printRelevantOptions includedOptions="log log-*" excludedOptions="log-console-* log-file log-file-* log-syslog-*">
-
-=== Console
-<@opts.includeOptions includedOptions="log-console-*"/>
-
-=== File
-<@opts.includeOptions includedOptions="log-file log-file-*"/>
-
-=== Syslog
-<@opts.includeOptions includedOptions="log-syslog-*"/>
+<@opts.printRelevantOptions includedOptions="log log-*" excludedOptions="log-console-* log-file log-file-* log-syslog-* log-mdc*">
```

Comment:
I just wonder, should we put some link or hint here that there are more options under the various log handlers? Folks tend to skip chapters and may miss there are log handlers with many more options.

Reply:
@vmuzikar That's a good point. I think it makes sense to include all config options related to the logging here.